src/components/templates/BottomSheetNotificationSettings/index.tsx

```diff
@@ -116,23 +116,23 @@ export const BottomSheetNotificationSettings = forwardRef<
       >
         <View style={styles.container}>
           {/* Price alerts */}
-          <Button
-            onPress={() =>
-              onSettingsValueChange(
-                'priceAlerts',
-                !localNotificationSettings.priceAlerts
-              )
-            }
-          >
-            <Row alignItems="center" justifyContent="space-between">
-              <Title>Price alert</Title>
+          <Row alignItems="center" justifyContent="space-between">
+            <Title>Price alert</Title>
+            <Pressable
```

Comment:
`onValueChange` prop of `Switch` can be used instead of `Pressable`

Reply:
also can we remove **disabled** prop of Switch than?